build/utils/scripts/dappnode_access_credentials.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# PRIORITY: Wi-Wi > Avahi > VPN (Wireguard > OpenVpn)
+
+#############
+#0.VARIABLES#
+#############
+# Containers
+WIFI_CONTAINER="DAppNodeCore-wifi.dnp.dappnode.eth"
+WIREGUARD_CONTAINER="DAppNodeCore-wireguard.dnp.dappnode.eth"
+OPENVPN_CONTAINER="DAppNodeCore-vpn.dnp.dappnode.eth"
+HTTPS_CONTAINER="DAppNodeCore-https.dnp.dappnode.eth"
+# Credentials
+WIREGUARD_CREDS="docker exec $WIREGUARD_CONTAINER cat /config/peer_dappnode_admin/peer_dappnode_admin.conf"
+OPENVPN_CREDS="docker exec -i $OPENVPN_CONTAINER getAdminCredentials"
+WIFI_CREDS=$(cat /usr/src/dappnode/DNCORE/docker-compose-wifi.yml 2> /dev/null | grep 'SSID\|WPA_PASSPHRASE')
+# Endpoints
+DAPPNODE_ENDPOINT="http://my.dappnode"
+DAPPNODE_AVAHI_ENDPOINT="http://my.dappnode.local"
+
+#############
+#1.FUNCTIONS#
+#############
+
+# $1 Connection method $2 Credentials
+function create_connection_message () {
+  echo -e "\e[32mConnect to DAppNode through $1 using the following credentials:\e[0m\n$2\nVisit \e[4m$DAPPNODE_ENDPOINT\e"
+}
+
+function wifi_connection () {
+  # NOTE: network interface may be in use by host => $(cat /sys/class/net/${INTERFACE}/operstate)" !== "up")
+  # NOTE: wifi has a delay up to 1 min
+  INTERFACE=$(ls /sys/class/ieee80211/*/device/net 2> /dev/null)
+  [ "$(docker inspect -f '{{.State.Running}}' ${WIFI_CONTAINER} 2> /dev/null)" = "true" ] && [ ! -z $INTERFACE ] && create_connection_message "Wi-Fi" "$WIFI_CREDS" && exit 0 || echo "Wifi not available"
```

Comment:
Is the `&& exit 0` necessary?

Reply:
It deppends: do we want to show every access credentials? (wifi-avahi-vpn)
If we only want to show credentials from the first service available based on priority, we should do `exit 0` when the access method is available

Reply:
See comment about `return` and add a comment explaining the need of exit 0 in the code